parquet-variant-compute/src/lib.rs

```diff
@@ -20,10 +20,14 @@
 //! ## Main APIs
 //! - [`VariantArray`] : Represents an array of `Variant` values.
 //! - [`VariantArrayBuilder`]: For building [`VariantArray`]
-//! - [`json_to_variant`]: Function to convert a batch of JSON strings to a `VariantArray`.
-//! - [`variant_to_json`]: Function to convert a `VariantArray` to a batch of JSON strings.
-//! - [`mod@cast_to_variant`]: Module to cast other Arrow arrays to `VariantArray`.
-//! - [`variant_get`]: Module to get values from a `VariantArray` using a specified [`VariantPath`]
+//!
+//! # Compute Kernels
+//! - [`json_to_variant()`]: Function to convert Arrays of JSON strings to a `VariantArray`.
+//! - [`variant_to_json()`]: Function to convert a `VariantArray` to arrays of JSON strings.
+//! - [`cast_to_variant()`]: Cast Arrow arrays to `VariantArray`.
+//! - [`variant_get()`]: Convert `VariantArray` (or an inner path) to ArrowArrays type
```

Comment:
What is "ArrowArrays type", sorry?

```suggestion
//! - [`variant_get()`]: Convert `VariantArray` (or an inner path) to a strongly-typed Arrow array.
```